src/System.IO.Pipes/ref/System.IO.Pipes.Manual.cs

```diff
@@ -1,18 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-// ------------------------------------------------------------------------------
-// Changes to this file must follow the http://aka.ms/api-review process.
-// ------------------------------------------------------------------------------
-
-
-namespace System.IO.Pipes
-{
-    // It needs to be defined manually because we cannot rely on Stream's implementation due to lack of APM virtual methods
```

Comment:
I don't think this file is necessary anymore. Building for netcoreapp/uap only.

Reply:
Please correct me if I am mistaken.
https://github.com/dotnet/corefx/commit/40723b795917f2e98b64d45c17effb2546dd0cae#diff-d1b2704483ad3363a0efe36a0e5c64cfR18

cc @ericstj